code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -646,16 +650,18 @@
 
 /datum/gas_reaction/nitrium_formation/init_reqs()
 	requirements = list(
-		/datum/gas/tritium = 20,
-		/datum/gas/nitrogen = 10,
-		/datum/gas/bz = 5,
+		/datum/gas/tritium = MINIMUM_MOLE_COUNT,
+		/datum/gas/nitrogen = MINIMUM_MOLE_COUNT,
+		/datum/gas/bz = MINIMUM_MOLE_COUNT,
 		"MIN_TEMP" = NITRIUM_FORMATION_MIN_TEMP,
 	)
 
 /datum/gas_reaction/nitrium_formation/react(datum/gas_mixture/air)
 	var/list/cached_gases = air.gases
 	var/temperature = air.temperature
-	var/heat_efficency = min(temperature / NITRIUM_FORMATION_TEMP_DIVISOR, cached_gases[/datum/gas/tritium][MOLES], cached_gases[/datum/gas/nitrogen][MOLES], cached_gases[/datum/gas/bz][MOLES] * INVERSE(0.05))
+	var/minimal_mole_factor = min(cached_gases[/datum/gas/tritium][MOLES], cached_gases[/datum/gas/nitrogen][MOLES], cached_gases[/datum/gas/bz][MOLES] * INVERSE(0.05))
+	var/thermal_efficiency = 1 - NUM_E ** -((temperature - NITRIUM_FORMATION_MIN_TEMP) / NITRIUM_FORMATION_TEMP_DIVISOR)
+	var/heat_efficency = min(minimal_mole_factor * thermal_efficiency, cached_gases[/datum/gas/tritium][MOLES], cached_gases[/datum/gas/nitrogen][MOLES], cached_gases[/datum/gas/bz][MOLES] * INVERSE(0.05))
```

Comment:
and here

Reply:
Same as above, but for temperature. It's offset by the minimum temperature for formation, so it is easier to understand this as the difference between temperature and minimum formation temperature when you want to scale it.